components/chatbot/index.tsx

```diff
@@ -237,6 +238,7 @@ export default function Chatbot({ tenant, conversationId, initMessage, onSelecte
               sources={[]}
               onSelectedDocumentId={onSelectedDocumentId}
               model={pendingMessage?.model || selectedModel}
+              isGenerating={true}
```

Comment:
Nit: you don't need to include `{true}`

```suggestion
              isGenerating
```